test/basic.html

```diff
@@ -74,6 +74,22 @@
         }, 12);
       });
 
+      test('show() will accept only text and duration parameters', function() {
+        toast = fixture('basic');
+        toast.show({text: 'hello world', duration: 20});
+        assert.isTrue(toast.opened, '`opened` is true');
+        assert.equal(toast.text, 'hello world', '`text` is correct');
+        assert.equal(toast.duration, 20, '`duration` is correct');
+        toast.show({foo: 'bar'});
+        assert.isUndefined(toast.foo, '`foo` is not a valid propery and will not be set');
+      });
+
+      test('show() will accept a string argument as the text parameter', function() {
+        toast = fixture('basic');
+        toast.show('hello world 2');
+        assert.equal(toast.text, 'hello world 2', '`text is correct`');
```

Comment:
to be safe, let's also check that the toast was actually opened ;)
`assert.isTrue(toast.opened, '`opened` is true');`